FAQ.md

```diff
@@ -104,10 +104,10 @@ dependency, you have a couple of options:
 2. Use an override.
 
 Overrides are a sledgehammer, and should only be used as a last resort. While
-dependencies and overrides are declared in the same way in `Gopkg.toml`, they
+constraints and overrides are declared in the same way in `Gopkg.toml`, they
 behave differently:
 
-* Dependencies:
+* Constraints:
    1. Can be declared by any project's manifest, yours or a dependency
    2. Apply only to direct dependencies of the project declaring the constraint
    3. Must not conflict with the `dependencies` declared in any other project's manifest
```

Comment:
Oops! There's one more that needs to be updated from `dependencies` to `[constraint]`'s. Not 100% sure how to refer to the header in the plural form.